Cargo.toml

```diff
@@ -12,17 +12,16 @@ edition = "2018"
 
 [features]
 mock = ["mockito"]
-time = ["chrono"]
+time = []
```

Comment:
Shouldn't this be removed since the feature is not used in the code anymore?

Reply:
This feature is used to define a DataTime in `types::response` module

Reply:
But why, shouldn't all be `chrono`?

Reply:
🤔

Reply:
Why this module still uses cfg with String/chrono, shoudn't all be chrono now?

Reply:
I think having String in `response` module is somehow interesting, chrono is used to define time ranges as input, but output seems like something optional.

Reply:
it was this PR https://github.com/naomijub/transistor/pull/31

Reply:
related docs are https://github.com/naomijub/transistor#using-chrono